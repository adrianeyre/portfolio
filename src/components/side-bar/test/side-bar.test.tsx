import { shallow } from 'enzyme';

import ISideBarProps from '../interface/side-bar-props';
import SideBar from '../side-bar';

describe('SideBar', () => {
	const mockRouteProp: any = jest.fn();

	const defaultProps: ISideBarProps = {
		data: undefined,
		linksData:  undefined,
		scrollToAnchor: mockRouteProp,
		showModal: mockRouteProp,
	};

	it('Should render correctly', () => {
		const sidebarComponent = shallow(<SideBar { ...defaultProps }/>);

		expect(sidebarComponent).toMatchSnapshot();
	});
});
