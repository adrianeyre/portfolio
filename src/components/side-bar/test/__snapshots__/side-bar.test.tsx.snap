// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideBar Should render correctly 1`] = `ShallowWrapper {}`;
